playbooks/common/openshift-cluster/config.yml

```diff
@@ -8,10 +8,10 @@
   tags:
   - always
   gather_facts: no
-  tasks:
-  - include_role:
-      name: openshift_excluder
-      tasks_from: disable
+  roles:
+  - role: openshift_excluder
+    r_openshift_excluder_action: disable
+    r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
 
```

Comment:
I would like to discuss include_role vs roles with the team.  include_role seems much more useful, to me, than role. I'd like to further this discussion.